### ISSUE TITLE
Log spam: [Extreme Sound Muffler/]: ESM: Muffled list not found; Creating empty Muffled list

### ISSUE THREAD

**Issue by aaronhowser1** [LeoBeliik/ExtremeSoundMuffler#66]
### Extreme sound muffler version

extremesoundmuffler-3.30_forge-1.18.2

### Forge / Fabric version

40.2.14

### Describe the bug

![image](https://github.com/LeoBeliik/ExtremeSoundMuffler/assets/15698722/f8118b3c-08ae-4b98-ba81-617dd1a7ffeb)

I keep getting that spammed. Strangely enough, it always happens after my KubeJS script (which is triggered by the EntityJoinWorldEvent) logs something. Maybe it's getting triggered by the same event?

### Crashlog

https://gist.github.com/aaronhowser1/a34a826cc3d1ec7c8a6ba0eb23c9087f

### Additional context

The KubeJS script:

```js
if (Platform.mods.sdrp) {
    onForgeEvent(
        'net.minecraftforge.event.entity.EntityJoinWorldEvent',
        (event) => {
            if (!event.getWorld().isClientSide()) return
            if (event.getEntity().type !== 'entity.minecraft.player') return
            const dimPath = event.getWorld().dimension().location().getPath()
            console.log(dimPath)
            SDRP.setState(`sdrp.${dimPath}.in`, `sdrp.${dimPath}`, 'dimPath')
        }
    )
}
```

**Comment by aaronhowser1** [comment]
![image](https://github.com/LeoBeliik/ExtremeSoundMuffler/assets/15698722/e49ac120-3f52-4b94-80d2-412037200efd)
It also doesn't actually create the file. I don't have an /ESM/ folder at all

**Comment by LeoBeliik** [comment]
Yes, this was fixed in new versions, it just never landed to 1.18 and unfortunately I can't backport it.
It doesn't create a folder because it doesn't need to, which makes the log spamming everytime a player joins the world, to fix it simply open and close the muffler screen once and it should create the folder and files it needs to stop spamming this.
Sorry for that!
